weaver.go

```diff
@@ -210,6 +210,15 @@ func runRemote[T any, _ PointerToMain[T]](ctx context.Context, app func(context.
 	return <-errs
 }
 
+// WeaverInfo holds runtime information about a deployed application.
+type WeaverInfo struct {
+	// Unique identifier for the application deployment.
+	DeploymentID string
+
+	// TODO(spetrovic): Add other runtime fields here (e.g., application start
+	// time, name of the deployer).
+}
+
```

Comment:
If you want to check that `WeaverInfo` and `weaver.WeaverInfo` have the same underlying type (ignoring struct tags), I think you can do the following:

```go
var (
    _ = WeaverInfo(weaver.WeaverInfo{})
    _ = weaver.WeaverInfo(WeaverInfo{})
)
```

https://go.dev/ref/spec#Conversions describes how this conversion is legal only if the two types (i.e. `WeaverInfo` and `weaver.WeaverInfo`) have the same underlying type.

Reply:
Great suggestion, done!